app/src/sharedTest/java/org/oppia/app/help/HelpFragmentTest.kt

```diff
@@ -50,7 +50,7 @@ class HelpFragmentTest {
   }
 
   @Test
-  fun openHelpActivity_configurationChanged_scrollRecyclerViewToZeroPosition_showsFAQSuccessfully() {
+  fun openHelpActivity_configurationChanged_scrollRecyclerViewToZeroPosition_showsFAQSuccessfully() { // ktlint-disable max-line-length
```

Comment:
Bring this comment `// ktlint-disable max-line-length` in new line.

Reply:
We cannot do that because ktlint will check that line only in which this comment is written which is more than 100 char

Reply:
Oh alright.